### ISSUE TITLE
Document the process for configuring logging from a file

### ISSUE THREAD

**Issue by tintoy** [tintoy/seqlog#4]
Propose using `logging.config.dictConfig` (see https://docs.python.org/3/library/logging.config.html for implementation details).

Add a section to the docs for configuring logging from a file. Content should be something like:

```python
import logging
import yaml

def configure_from_file(file_name):
  """
  Convenience function to configure logging from a YAML-format configuration file.
  """

  with open('./logging-config.yml') as logging_config_file:
    logging_config = yaml.load(logging_config_file)

  logging.config.dictConfig(logging_config)
```

Where `logging-config.yml` contains:

```yaml
# Configure logging from scratch.
disable_existing_loggers: True

# Configure the root logger to use Seq
root:
  class: seqlog.structured_logging.StructuredLogger
  level: INFO
  handlers:
    - seq

handlers:
  seq:
    class: seqlog.structured_logging.SeqLogHandler
    formatter: seq

    # Seq-specific settings (add any others you need, they're just kwargs for SeqLogHandler's constructor).
    server_url: 'http://localhost:5341/'
    api_key: 'MyApiKey'

formatters:
  seq:
    style: '{'
```

Addresses tintoy/seqlog/#3.

**Comment by tintoy** [comment]
Create a wrapper for the following logic:

```python
from logging.config import dictConfig
import yaml

def configure_from_file(file_name):
  with open('./logging-config.yml') as logging_config_file:
    logging_config = yaml.load(logging_config_file)

    dictConfig(logging_config)
```

Where `logging-config.yml` contains:

```yaml
# Configure logging from scratch.
disable_existing_loggers: True

# Configure the root logger to use Seq
root:
  class: seqlog.structured_logging.StructuredLogger
  level: INFO
  handlers:
    - seq

handlers:
  seq:
    class: seqlog.structured_logging.SeqLogHandler
    formatter: seq

    # Seq-specific settings (add any others you need, they're just kwargs for SeqLogHandler's constructor).
    server_url: 'http://localhost:5341/'
    api_key: 'MyApiKey'

formatters:
  seq:
    style: '{'
```

**Comment by tintoy** [comment]
Implemented as part of #3.
